rero_ils/modules/ill_requests/cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click command-line interface for ill_request record management."""
+
+from __future__ import absolute_import, print_function
+
+import json
+import random
+
+import click
+from flask.cli import with_appcontext
+
+from rero_ils.modules.ill_requests.api import ILLRequest
+from rero_ils.modules.items.cli import get_locations
+from rero_ils.modules.locations.api import Location
+from rero_ils.modules.patrons.api import Patron
+from rero_ils.modules.utils import get_ref_for_pid
+
+
+@click.command('create_ill_requests')
+@click.option('-f', '--requests_f', 'request_f', help='Request input file.')
```

Comment:
cosmetic: maybe better to use `request-file` instead `requests_f` and `request_f`